### PR TITLE
Start local storage plugin

### PR DESCRIPTION
**Pull request by smarlowucf**
Since implementation is pretty simple included it here.

### CODE REVIEW

**Comment by smarlowucf** [comment]
As discussed we normally use feature branch workflow. Internal developers have write access and can use this workflow. feature branch -> main. Developers can also choose to use a fork but that's only necessary for external contributors.

File: csp-billing-adapter-local.spec [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
```
Comment (rjschwei) [inline csp-billing-adapter-local.spec:18]: All the spec files should have
```
%define skip_python2 1
```
There are still projects in the build service that build against python2

File: csp-billing-adapter-local.spec [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+BuildRequires:  csp-billing-adapter
+%if %{with test}
+BuildRequires:  python3-pytest
+BuildRequires:  python3-coverage
+BuildRequires:  python3-pytest-cov
+%endif
+Requires:       python3-pluggy
+Requires:       csp-billing-adapter
+BuildArch:      noarch
+
```
Comment (rjschwei) [inline csp-billing-adapter-local.spec:37]: We want

```
%python_subpackages
```
Then when building against factory we get all the interpreters

File: csp-billing-adapter-local.spec [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+%build
+%python_build
+
+%install
+%python_install
+%python_expand %fdupes %{buildroot}%{$python_sitelib}
+
+%check
+%if %{with test}
+python3 -m pytest --cov=csp_billing_adapter_local
```
Comment (rjschwei) [inline csp-billing-adapter-local.spec:53]: That would run the tests only against the default interpreter. Which is probably generally sufficient. However when building for multiple interpreters the modules for the default interpreter may not be visible. This should use a macro expansion

**Pull request merged**